scripts/configure-to-bundle.js

```diff
@@ -34,6 +34,9 @@ packageJson.bundledDependencies = [];
 for(const x in packageJson.dependencies) {
     packageJson.bundledDependencies.push(x);
 }
+for(const x in (packageJson.optionalDependencies || {})) {
+    packageJson.bundledDependencies.push(x);
+}
```

Comment:
I probably missed it, but I didn't see this in the PR description. 😋 
Makes sense to bundle the optional dependencies too.

Reply:
We needed this now since Keytar is an optional dependency of `@zowe/cli@next` 🙂